api/src/api/services/moonbeam/balance.ts

```diff
@@ -75,12 +75,12 @@ export const fundMoonbeamEphemeralAccount = async (ephemeralAddress: string) =>
   }
 };
 
-export function getMoonbeamFundingData(decimals: number): {
+export function getMoonbeamFundingData(decimals: number, boosted: boolean = false): {
   fundingAmountRaw: string;
   walletClient: ReturnType<typeof createMoonbeamClientsAndConfig>['walletClient'];
   publicClient: ReturnType<typeof createMoonbeamClientsAndConfig>['publicClient'];
 } {
-  const fundingAmountUnits = Big(MOONBEAM_EPHEMERAL_STARTING_BALANCE_UNITS);
+  const fundingAmountUnits = boosted ? Big(MOONBEAM_EPHEMERAL_STARTING_BALANCE_UNITS_ETHEREUM) : Big(MOONBEAM_EPHEMERAL_STARTING_BALANCE_UNITS);
```

Comment:
`boosted` reminds me of squidrouter. But that's not entirely true so let's change the name. Maybe call it `largeFunding: boolean = false`?